var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -94,6 +94,9 @@ class Abinit(AutotoolsPackage):
     # conflicts('+elpa', when='~mpi')
     # conflicts('+elpa', when='+scalapack')
 
+    patch('rm_march_settings.patch', when='%fj')
+    patch('fix_for_fujitsu.patch', level=0, when='%fj')
```

Comment:
Two questions:
1. Is there any url from which we can pull this patch?
2. If not, was this patch contributed upstream, or is there an issue that can be referred in a comment?

Reply:
Please wait for a while as we will check with the person in charge of correction.

Reply:
@alalazo 
Sorry for the late response.
This patch cannot be pulled from anywhere. This is a local patch limited to the Fujitsu compiler environment.
Therefore, it is difficult to apply it to upstream as it is, so we are checking in the following issue whether it can be fixed more generically.
https://github.com/abinit/abinit/issues/43
First of all, can Spack allow me to apply this patch to previous versions?